<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>721</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Login">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLabel" name="loginLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>Enter your name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="loginTextEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="loginPushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>18</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Self">
     <attribute name="title">
      <string>My Profile</string>
     </attribute>
     <widget class="QTableWidget" name="profileFriendsTable">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>250</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QLabel" name="profilePostsLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>541</width>
        <height>151</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="profileTitleLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>My Profile</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="profileSuggestionsTable">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>200</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>430</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Polls</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>My Posts</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="profilePollTable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>440</y>
        <width>461</width>
        <height>151</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QSpinBox" name="profilePollSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>480</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Other">
     <attribute name="title">
      <string>Browse</string>
     </attribute>
     <widget class="QPushButton" name="otherAddButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>+ Add Friend</string>
      </property>
     </widget>
     <widget class="QLabel" name="otherTitleLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>Add some friends first!</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="otherPostsLabel">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>471</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="otherFriendsTable">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>250</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QTableWidget" name="otherPollTable">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>351</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Polls</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Latest Posts</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Their Friends</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="otherPollSpinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>460</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Post">
     <attribute name="title">
      <string>+ Post</string>
     </attribute>
     <widget class="QTextEdit" name="postEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string extracomment="Add a post..."/>
      </property>
     </widget>
     <widget class="QPushButton" name="postButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Post &gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="pollTable">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>250</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <column/>
     </widget>
     <widget class="QLabel" name="postLabel1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create Post</string>
      </property>
     </widget>
     <widget class="QLabel" name="pollLabel1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create Poll</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pollButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>520</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Post &gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pollAddOptionButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>490</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Add Option</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="pollMessageEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pollLabel2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>350</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="pollOptionEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>440</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pollRemoveOptionButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>520</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Remove Option</string>
      </property>
     </widget>
     <widget class="QLabel" name="pollLabel3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Option:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="postIncomingEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="postLabel2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nunito Sans</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Message:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="postIncoming">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Incoming Post?</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nunito Sans</family>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Democracy Network</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
